src/organisms/organism.py

```diff
@@ -24,11 +25,14 @@ def update_lifetime(self):
     def draw(self, config, display):
         pygame.draw.circle(display, (255, 0, 0), (self.position.x * config.tile_width, self.position.y * config.tile_width), config.tile_width / 2)
 
-    def checktile(self, world):
-        tile_type = world[self.position.y][self.position.x].type
-        if tile_type == "empty":
-            ...
-        elif tile_type == "food":
-            ...
-        elif tile_type == "danger":
-            ...
+    def vision(self):
+         view = [World.grid[self.position.y - 2][self.position.x - i for i in range ( -2, 3)].content, 
+            World.grid[self.position.y - 1][self.position.x - i for i in range ( -2, 3)].content, 
+            World.grid[self.position.y][self.position.x - i for i in range ( -2, 3)].content,
+            World.grid[self.position.y + 1][self.position.x - i for i in range ( -2, 3)].content,
+            World.grid[self.position.y + 2][self.position.x - i for i in range ( -2, 3)].content]
```

Comment:
- "world" kleinschreiben: Klassen werden gross geschrieben, aber hier ist world ein Objekt (eine Instanz der Klasse "World")
- Wenn ich das richtig verstanden habe hast du ein 5x5 quadrat gemacht. Das kannst du am einfachsten lösen, in dem du 2 for Loops "verschachtelst", der äussere geht über die Reihen, der innere für jede Reihe über die Spalten.
Ich habe hier einen entsprechenden Anpassungsvorschlag:

```suggestion
        view = []
        for row in range(-2, 3):
            for column in range(-2, 3):
                view.append(world.grid[self.position.y + row][self.position.x + column].content)
```